bin/citrea/tests/test_helpers/mod.rs

```diff
@@ -33,6 +34,7 @@ pub enum NodeMode {
 #[allow(clippy::too_many_arguments)]
 pub async fn start_rollup(
     rpc_reporting_channel: oneshot::Sender<SocketAddr>,
+    utility_server_reporting_channel: Option<oneshot::Sender<SocketAddr>>,
```

Comment:
IMO, RPC & Utility server channels should be passed as last params and not first. Contextually `start_rollup` called by passing some secondary values does not provide the best clarify on what this does.